src/components/Banner.jsx

```diff
@@ -0,0 +1,16 @@
+import styled, { css } from "styled-components";
+
+export const Banner = styled.div`
```

Comment:
I know it's not this PR that does it, but I find the change banner to be hard to read. Thoughts? 
![image](https://user-images.githubusercontent.com/43038510/215502881-0112fc74-db5e-4a86-a983-bd137bae4dcb.png)

Reply:
Ok sounds good. resolving